azkaban-exec-server/src/main/java/azkaban/execapp/reporter/AzkabanKafkaAvroEventReporter.java

```diff
@@ -0,0 +1,121 @@
+package azkaban.execapp.reporter;
+
+import azkaban.event.Event;
+import azkaban.utils.Props;
+import com.google.common.base.Preconditions;
+import gobblin.metrics.MetricContext;
+import gobblin.metrics.event.EventSubmitter;
+import gobblin.metrics.kafka.KafkaAvroEventReporter;
+import gobblin.metrics.kafka.KafkaAvroSchemaRegistry;
+import gobblin.metrics.kafka.KafkaEventReporter;
+import gobblin.metrics.kafka.KafkaPusher;
+import java.util.Map;
+import java.util.Properties;
+import org.apache.log4j.Logger;
+
+
+/**
+ * Default implementation of the <code>AzkabanEventReporter</code> class.
+ * An instance of this class is injected via guice into the <code>FlowRunnerManager</code>
+ * class when event reporting is enabled.
+ */
+public class AzkabanKafkaAvroEventReporter implements AzkabanEventReporter {
+
+  private static final Logger logger = Logger.getLogger(AzkabanKafkaAvroEventReporter.class);
+  private static final String EVENT_REPORTING_KAFKA_BROKERS = "event.reporting.kafka.brokers";
```

Comment:
Also, we start to prefix the keys with Azkaban to make it clear that these keys are defined by Azkaban instead of defined by plugin code.